docs/sophia_stdlib.md

```diff
@@ -754,12 +880,20 @@ Verifies a standard 64-byte ECDSA signature (`R || S`).
 
 ### Int
 
-#### to_str
+#### to\_str
+```
+Int.to_str(n : int) : string
 ```
-Int.to_str : int => string
+
+Casts the integer to a string (in decimal representation).
+
+#### to\_bytes
+```
+Int.to_bytes(n : int, size : int) : bytes()
 ```
 
-Casts integer to string using decimal representation
+Casts the integer to a byte array with `size` bytes (big endian, truncating if
```

Comment:
How are negative numbers marked? Is there non-truncated version?

(i have some perverted urge for a dependent fixed-size version too, but maybe later)

Reply:
`Int.to_bytes(x : int) : bytes(n)` would be an option, yes... But you can work around that. 

There is no non-truncated version, you'll have to do the math. The truncation will not preserve signedness, I'll make that specific in the docs. I.e. if you try to squeeze (-129) into a single byte that will be indistinguishable from 127.